microprofile/WFLY-12770_Create_quickstart_for_MicroProfile_Fault_Tolerance.adoc

```diff
@@ -0,0 +1,78 @@
+= WFLY-12770 Create quickstart for MicroProfile Fault Tolerance
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          quickstarts,microprofile,fault-tolerance
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The goal is to add a new Quickstart demonstrating how to make use of https://microprofile.io/project/eclipse/microprofile-fault-tolerance[Eclipse MicroProfile Fault Tolerance]
```

Comment:
I suggest to mention that this quickstart is based on https://quarkus.io/guides/microprofile-fault-tolerance.

Reply:
Done.